yente/search/indexer.py

```diff
@@ -76,7 +85,7 @@ async def index_entities_rate_limit(
 async def index_entities(es: AsyncElasticsearch, dataset: Dataset, force: bool) -> bool:
     """Index entities in a particular dataset, with versioning of the index."""
     if not dataset.load:
-        log.debug("Dataset is going to be loaded", dataset=dataset.name)
+        log.debug("Dataset is not going to be loaded", dataset=dataset.name)
```

Comment:
lol --- thanks for fixing it all up :)

Reply:
Haha, I spent a good 10 minutes being very confused by our log messages :D